_extras/checkout.md

```diff
@@ -49,40 +49,52 @@ and must be requested before the three months are up. To request an extension, p
 ## Part 1: Submit a Small Contribution to One of Our Lessons
 <a name="lesson-change"></a>
 
-Our lessons are maintained and improved by the people who teach them,
-so this part of the checkout procedure gives you a chance to familiarize yourself
+Our lessons and lesson infrastructure are maintained and improved by the people who use them.
+With this part of the checkout procedure, you will familiarize yourself
 with our collaborative curriculum development process.
 
-Trainees must make a contribution to a lesson's content, exercises, or instructor's guide by doing one of the following:
+### Examples of Contributions
 
-1. Providing substantive feedback on an existing issue or pull request (preferred).
-2. Submitting a change to fix an existing issue.
-3. Proof-reading a lesson and adding a new issue describing something to be improved.
+Trainees must make a contribution to any GitHub repository in The Carpentries organization that is actively maintained and supports a lesson (e.g. [LC Open Refine](https://github.com/LibraryCarpentry/lc-open-refine/), [SWC Unix Shell](http://swcarpentry.github.io/shell-novice/)) or lesson infrastructure (e.g. the [handbook](https://github.com/carpentries/handbook), [workshop template](https://github.com/carpentries/workshop-template)). 
 
-Contributions to Data Carpentry and Library Carpentry materials may be submitted through GitHub or by [email](mailto:{{ site.email }}). Contributions to Software Carpentry materials must be submitted through GitHub.
+We **encourage** the following types of contributions:
 
-Each lesson has a CONTRIBUTING.md file in its repository on GitHub. This file explains best practices for contributing
-to that lesson. You can access a lesson's GitHub repository by clicking on the GitHub kitty icon in the "Repository" column for that lesson on the lesson page. The lesson pages can be found at:  
+1. Comments on an existing issue or pull request (**preferred**).
+1. New pull requests the resolve an existing issue. 
+1. New issues describing something to be improved (e.g. reword a learning objective or the instructor notes)
+1. New issues describing something to be fixed (e.g. typos). 
```

Comment:
As a maintainer, my preference would be that typos are just fixed through a PR directly - an Issue for a typo creates more work all around IMO - not sure if I'm alone on this though!

Reply:
I'm always pleased to see issues that highlight text that is problematic in some way - whether that's about the language used or simply fails to communicate effectively - I'd prefer to see new issues for this type of problem - although maybe that falls under 'something to be improved' rather than 'something to be fixed'